ejb/src/main/java/biz/karms/sinkit/ejb/CoreService.java

```diff
@@ -16,6 +16,8 @@
 
     int getIocActiveHours();
 
+    void setIocActiveHours(int iocActiveHours);
```

Comment:
This wouldn't be needed if I could somehow set the IOC_ACTIVE_HOURS_ENV environment variable in a @Before method

then iocActiveHours would be Integer.parseInt(System.getenv(IOC_ACTIVE_HOURS_ENV));

Having iocActiveHours  assigned is, however, necessary, because it is being used in the tested method.

Reply:
@hugeox What is the problem with setting it programatically in @before?